ext/gmp/gmp.c

```diff
@@ -338,6 +289,39 @@ static zend_object *gmp_clone_obj(zend_object *obj) /* {{{ */
 }
 /* }}} */
 
+typedef void (*gmp_binary_op_t)(mpz_ptr, mpz_srcptr, mpz_srcptr);
+
+static zend_result binop_operator_helper(gmp_binary_op_t gmp_op, zval *return_value, zval *op1, zval *op2) {
+	mpz_ptr gmp_op1, gmp_op2, gmp_result;
+	if (!gmp_zend_parse_arg_into_mpz_ex(op1, &gmp_op1, 1, true)) {
+		if (!EG(exception)) {
+			zend_type_error("Number must be of type GMP|string|int, %s given", zend_zval_value_name(op1));
+		}
+		return FAILURE;
+	}
+	if (!gmp_zend_parse_arg_into_mpz_ex(op2, &gmp_op2, 1, true)) {
```

Comment:
You pass arg_num==1 here, is that right? shouldn't it be 2?

Reply:
It should yes